Makefile

```diff
@@ -126,8 +126,10 @@ endif
 
 else
 LDFLAGS += -rdynamic
+ifneq ($(OS),OpenBSD)
```

Comment:
The patches on the ports tree filters -lrt afterward (after all the `ifeq $($(CONFIG), ...)`), while this avoids to add it.

I tried to stay close to your way of doing, feel free to ask me to change as per your liking here.